server/core/src/https/views/login.rs

```diff
@@ -59,7 +60,7 @@ impl fmt::Display for ReauthPurpose {
         }
     }
 }
-
+#[derive(Clone)]
```

Comment:
Why do these need clone?

Reply:
Because `LoginDisplayCtx` has `domain_info` [here](https://github.com/kanidm/kanidm/pull/3352/files#diff-7886cf3d77b5287a765c6414392df5fa1e66ce34f033459147258dfc7df1af38R239) and the error from that needs to be able to access domain_info...